x-pack/plugin/core/src/main/java/org/elasticsearch/license/LicenseService.java

```diff
@@ -135,6 +137,9 @@ public class LicenseService extends AbstractLifecycleComponent implements Cluste
      */
     private final List<License.LicenseType> allowedLicenseTypes;
 
+    private final StartTrialClusterTask.Executor startTrialExecutor = new StartTrialClusterTask.Executor();
+    private final StartBasicClusterTask.Executor startBasicExecutor = new StartBasicClusterTask.Executor();
```

Comment:
NB by re-using these executors here we add support for batching of these tasks. Doesn't seem very important for these specific tasks, but as a rule we should avoid unbatched tasks.